app/scripts/modules/core/delivery/details/executionDetails.controller.js

```diff
@@ -11,7 +11,30 @@ module.exports = angular.module('spinnaker.executionDetails.controller', [
 
     controller.standalone = $scope.standalone || false;
 
+    function getStageParams(stageId) {
+      let summaries = ($scope.execution.stageSummaries || []);
+      let stageIndex = summaries.findIndex(s => (s.stages || []).some(s2 => s2.id === stageId));
+      if (stageIndex !== -1) {
+        let stepIndex = (summaries[stageIndex].stages || []).findIndex(s => s.id === stageId);
```

Comment:
Yea, we're not good about using `let` vs `const` "correctly."  Thankfully, it's not a lint rule that I had to turn off.